recipes/leptonica/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "leptonica" %}
+{% set version = "1.76.0" %}
+{% set sha256 = "ff98ffef6103cda0e400676f81113a8157569ac0f453c6e03a1a27b305ba25a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DanBloomberg/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cmake
+    - gcc       # [linux]
+    - libwebp 
+  run:
+    - zlib
+    - jpeg
+    - libpng
+    - libtiff
```

Comment:
These likely all need to be in `host` as well. C/C++ code compiles libraries that link to libraries. So they have to be around during the build for the linking part to occur. They also need to be included at run time with matching versions (we take care of that for you under the hood). This differs from Python where they just need to be around whenever an `import` line calls for them. Hope that helps.